src/main/java/edu/greenblitz/robotName/commands/swerve/MoveToAmp.java

```diff
@@ -0,0 +1,11 @@
+package edu.greenblitz.robotName.commands.swerve;
+
+import edu.greenblitz.robotName.Field;
+import edu.greenblitz.robotName.commands.auto.MoveToPose;
+
+public class MoveToAmp extends SwerveCommand{
+    @Override
+    public void initialize() {
+        MoveToPose.getPathCommand(Field.ScoringPositions.ampScorePosition).schedule();
+    }
+}
```

Comment:
Yoni - scheduling in the initialize of a command is very ugly, you can probably change MoveToPose so you can extend it to move to the amp position.